ui/packages/catalog-ui-search/src/main/webapp/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Entry from './js/Entry'
+
+Entry()
```

Comment:
Is `export` not needed here?

Reply:
It's unnecessary because this we don't expect anyone to pull in anything from this file.